src/packet_analysis/protocol/pbb/PBB.h

```diff
@@ -0,0 +1,31 @@
+// See the file "COPYING" in the main distribution directory for copyright.
+
+#pragma once
+
+#include "zeek/packet_analysis/Analyzer.h"
+#include "zeek/packet_analysis/Component.h"
+
+#define PBB_LEN 18u
```

Comment:
These should be defined like `constexpr int PBB_LEN = 18` instead of `#defines`. They should also be defined in the cc file since they're only used there.